apicurito/deploy/bundle/container.yaml

```diff
@@ -1,6 +1,6 @@
 annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: apicurito-v0.2
-  operators.operatorframework.io.bundle.channels.v1: apicurito-v0.2
+  operators.operatorframework.io.bundle.channel.default.v1: apicurito-v7.8
+  operators.operatorframework.io.bundle.channels.v1: apicurito-v7.8
```

Comment:
channel should be `apicurito-7.8.x`

Reply:
See https://docs.google.com/document/d/11wV1H4a1kPdz6Mkq7DSLqkncKqOpeIiDHhyTx2QvG0s/edit

Reply:
Fixed in deploy/bundle/metadata/annotations.yaml and deploy/bundle/Dockerfile too. Thanks.